OsmAnd-java/src/main/java/net/osmand/binary/RouteDataObject.java

```diff
@@ -368,7 +368,11 @@ public String getDestinationName(String lang, boolean transliterate, boolean dir
 				destinationTagLang = "destination:lang:" + lang;
 			}
 			String destinationTagDefault = "destination";
+			String destinationLanesTag = "destination:lanes";
```

Comment:
Original idea, not to use destination:lanes but to use only destination on next roads.  Check code here https://github.com/osmandapp/OsmAnd/blob/8ce37e4d8fcfad83b9127f5c24ab089d483e12e3/OsmAnd/src/net/osmand/plus/routing/RouteCalculationResult.java#L404